src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,43 @@
+@mixin hover($property, $toValue) {
+  transition: #{$property}, $effect-duration;
```

Comment:
```suggestion
  transition: #{$property}, $effect-duration ease;
```
something went wrong with transition.
try to set ease function.

devtools in demo-link
![Screenshot from 2023-09-17 16-38-31](https://github.com/mate-academy/react_phone-catalog/assets/33029864/b4692b87-d74d-46b2-b083-4774274e8e1c)
and after change transition property
![Screenshot from 2023-09-17 16-39-56](https://github.com/mate-academy/react_phone-catalog/assets/33029864/423c1f13-b651-4f09-b8b8-5ae1ec88771d)
![chrome-capture-2023-8-17 (1)](https://github.com/mate-academy/react_phone-catalog/assets/33029864/20a4942d-cf0a-437a-947f-d770ca4be354)

also can be problem with css imports. try to import without `./` in glonal-imports.